packages/gatsby-graphiql-explorer/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "gatsby-graphiql-explorer",
```

Comment:
this probably need better package name - this is kind of express middleware, but not exactly, because it's not used as `express.use(someMiddleware())`, instead it is used as `graphiqlExplorer(expressApp, params)` as I need to have 2 route handlers (one for html and one for bundle)

Reply:
It's easy to bike shed here--I don't really have a problem with the name. We could also scope it, e.g. `@gatsbyjs/graphiql` which is really what we've done here. We tweaked `graphiql` with some extra functionality.